kernel/constr.ml

```diff
@@ -72,10 +72,22 @@ type case_info =
 type 'constr pexistential = existential_key * 'constr array
 type ('constr, 'types) prec_declaration =
     Name.t array * 'types array * 'constr array
+
 type ('constr, 'types) pfixpoint =
     (int array * int) * ('constr, 'types) prec_declaration
+  (* The array of [int]'s tells for each component of the array of
```

Comment:
Can you move this comment to the `.mli` file? It really describes the API rather than internal implementation.

Reply:
Done